scripts/lint_translations.clj

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bb
+
+(ns lint-translations
+  (:require [babashka.pods :as pods]))
+
+(pods/load-pod 'clj-kondo/clj-kondo "2023.09.07")
+(require '[pod.borkdude.clj-kondo :as kondo])
+(require '[cheshire.core :as json])
+(require '[clojure.set :as set])
+
+(def src-paths ["src"])
+
+(def translation-file "translations/en.json")
+
+(defn- safe-name
+  [x]
+  (when x (name x)))
+
+(defn- ->keyword
+  [analysis-keyword]
+  (keyword (safe-name (:ns analysis-keyword)) (:name analysis-keyword)))
+
+(defn- report-issues
+  [incorrect-usages]
+  (doseq [incorrect-usage incorrect-usages]
+    (->> incorrect-usage
+         ((juxt :filename :row :reason ->keyword))
+         (apply format "%s:%s %s %s")
+         println)))
+
+(defn- extract-translation-keys
+  [file]
+  (-> file slurp json/parse-string keys))
+
+
+(def ^:private possibly-unused-warning
```

Comment:
I've got tons of messages "Possibly Unused ...", which kind of pollutes the output from `make lint` substantially.

Also, some (I haven't checked all) are really not used, like `whats-on-your-mind`, but we're flagging this translation as *possibly unused*.

Could you explain why this is happening? And if we can fix?

Reply:
Please see my [earlier comment](https://github.com/status-im/status-mobile/pull/17735#issuecomment-1792343874) about this.